### ISSUE TITLE
run success, but page show errors

### ISSUE THREAD

**Issue by HCLQ** [vuejs/vuepress#1835]
- [x] I confirm that this is an issue rather than a question.

#### Steps to reproduce
in lerna project, make a docs dir with README.md
run command:
```
yarn add vuepress -W -D
npm run docs:dev     // vuepress dev docs
```

#### What is expected?
cli show success and open http://localhost:8080/ to view the page
#### What is actually happening?
the page show errors like
```
  Error: webpack-internal:///./node_modules/@vuepress/core/node_modules/webpack/buildin/harmony-module.js:34
  /* WEBPACK VAR INJECTION */}.call(this, __webpack_require__(/*! ./harmony-module.js */ "./node_modules/@vuepress/core/node_modules/webpack/buildin/harmony-module.js")(module)))//# sourceU  RL=[module]
                                                                                                                                                                        ^
  TypeError: __webpack_require__(...) is not a function
  
  - harmony-module.js:34 eval
    [.]/[core]/[webpack]/buildin/harmony-module.js:34:167
  
  - index.dev.html:789 Module../node_modules/@vuepress/core/node_modules/webpack/buildin/harmony-module.js
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:789:1
  
  - index.dev.html:687 __webpack_require__
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:687:30
  
  - index.dev.html:61 fn
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:61:20
  
  - module.js:30 eval
    [.]/[core]/[webpack]/buildin/module.js:30:41
  
  - index.dev.html:801 Module../node_modules/@vuepress/core/node_modules/webpack/buildin/module.js
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:801:1
  
  - index.dev.html:687 __webpack_require__
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:687:30
  
  - index.dev.html:61 fn
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:61:20
  
  - lodash.js:17106 eval
    [.]/[core]/[lodash]/lodash.js:17106:41
  
  - index.dev.html:766 Object../node_modules/@vuepress/core/node_modules/lodash/lodash.js
    [ymm-plugin]/[@vuepress]/core/lib/client/index.dev.html:766:1
  
Html Webpack Plugin:
```
#### Other relevant information
```
Environment Info:

  System:
    OS: macOS 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz
  Binaries:
    Node: 12.8.0 - /usr/local/bin/node
    Yarn: 1.16.0 - ~/.yarn/bin/yarn
    npm: 6.10.2 - /usr/local/bin/npm
  Browsers:
    Chrome: 76.0.3809.132
    Firefox: Not Found
    Safari: 12.1.2
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0 
    @vue/babel-plugin-transform-vue-jsx:  1.0.0 
    @vue/babel-preset-app:  3.11.0 
    @vue/babel-preset-jsx:  1.1.0 
    @vue/babel-sugar-functional-vue:  1.0.0 
    @vue/babel-sugar-inject-h:  1.0.0 
    @vue/babel-sugar-v-model:  1.0.0 
    @vue/babel-sugar-v-on:  1.1.0 
    @vue/cli-shared-utils: ^3.9.0 => 3.9.0 
    @vue/component-compiler-utils:  3.0.0 
    typescript:  3.5.3 
    vue: ^2.6.10 => 2.6.10 
    vue-cli-plugin-ymm:  0.1.1 
    vue-hot-reload-api:  2.3.3 
    vue-loader:  undefined (15.7.1)
    vue-router:  3.1.2 (3.1.3)
    vue-server-renderer:  2.6.10 
    vue-style-loader:  4.1.2 
    vue-template-compiler:  2.6.10 
    vue-template-es2015-compiler:  1.9.1 
    vuepress: ^1.0.3 => 1.0.3 
    vuepress-html-webpack-plugin:  undefined (3.2.0)
    vuepress-plugin-container:  2.0.2 
    vuex:  3.1.1 
    vuex-router-sync:  5.0.0 
  npmGlobalPackages:
    @vue/cli: 3.1.3
```
- Output of `npx vuepress info` in my VuePress project:
```
Environment Info:

  System:
    OS: macOS 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz
  Binaries:
    Node: 12.8.0 - /usr/local/bin/node
    Yarn: 1.16.0 - ~/.yarn/bin/yarn
    npm: 6.10.2 - /usr/local/bin/npm
  Browsers:
    Chrome: 76.0.3809.132
    Firefox: Not Found
    Safari: 12.1.2
  npmPackages:
    @vuepress/core:  1.0.3 
    @vuepress/theme-default:  1.0.3 
    vuepress: ^1.0.3 => 1.0.3 
  npmGlobalPackages:
    vuepress: Not Found
```

**Comment by kefranabg** [comment]
Hi @HCLQ, thanks for the report.

Are you running Vuepress in a Vue.js repository? Is it possible that you provide a reproduction link?

**Comment by HCLQ** [comment]
> Hi @HCLQ, thanks for the report.
> 
> Are you running Vuepress in a Vue.js repository? Is it possible that you provide a reproduction link?

Yes, I'm writing docs in a vue project.
Now,i find the vuePress@0.14.8 is work well.

**Comment by kefranabg** [comment]
Could you try to remove `@vue/cli-service` this package from your project and run vuepress again? If it's working this issue is related to [this PR](https://github.com/vuejs/vuepress/pull/1771) that will fix this problem. (Will be released in VP `1.0.4`)
